atomate/common/powerups.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+
+"""
+This module defines general powerups that can be used for all workflows
+"""
+
+from atomate.utils.utils import get_fws_and_tasks
+from fireworks.core.firework import Workflow
+
+__author__ = "Janine George, Guido Petretto"
+__email__ = 'janine.george@uclouvain.be, guido.petretto@uclouvain.be'
+
+
+def set_queue_adapter(original_wf: Workflow, queueadapter: dict = None,

```

Comment:
I would really like to change every keyword in the queue_adapter. I am sometimes replacing complete lines in the job files. I don't think this is possible with the current set_queue_options. This would also result in a less flexible setup for job script templates if we introduced additional keywords to set_queue_options.